package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sanity-next-starter",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
Let's talk about adding in our standard rules for projects, so we ensure all code we write (regardless of project) is the same quality.

Reply:
Done in c80136d